com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -19,14 +19,11 @@ public class NetworkManager : MonoBehaviour, INetworkUpdateSystem, IProfilableTr
         // RuntimeAccessModifiersILPP will make this `public`
         internal static readonly Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>> __rpc_func_table = new Dictionary<uint, Action<NetworkBehaviour, NetworkSerializer, __RpcParams>>();
 
-#if UNITY_EDITOR || DEVELOPMENT_BUILD
-
+#if DEVELOPMENT_BUILD || UNITY_EDITOR
         // RuntimeAccessModifiersILPP will make this `public`
         internal static readonly Dictionary<uint, string> __rpc_name_table = new Dictionary<uint, string>();
-#else // !(UNITY_EDITOR || DEVELOPMENT_BUILD)
-        // RuntimeAccessModifiersILPP will make this `public`
-        internal static readonly Dictionary<uint, string> __rpc_name_table = null; // not needed on release builds
```

Comment:
we kept this field for the old ILPP implementation (pre-2020.3 ILPP, targeting 2019.4) because we needed it for `nameof(NetworkManager.__rpc_name_table)` code.
however, ILPP code is refactored (cleaned) on 2020.3+ and now we don't actually require this field to be there in the middle of ILPP.
so, let's get rid of it and wrap lines using `__rpc_name_table` with `#if DEVELOPMENT_BUILD || UNITY_EDITOR` everywhere!